clients/src/main/java/com/school/clients/finance/StudentFinanceClient.java

```diff
@@ -14,4 +17,8 @@
 
     @PostMapping("/api/finance/students")
     StudentFinanceRegisterResponse register(@RequestBody StudentFinanceRegisterRequest request);
+
+    @GetMapping("/api/finance/students/{studentId}")
```

Comment:
did you test this I think this is not work! because do not put / before studentId in controller